sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -381,12 +468,24 @@ def from_predictions(
             y_true, y_pred, pos_label=pos_label, sample_weight=sample_weight
         )
 
+        if plot_chance_level:
+            pos_prevalence = Counter(y_true)[pos_label] / len(y_true)
+        else:
+            pos_prevalence = None
```

Comment:
Let's always compute the prevalence. Like this even if you pass `plot_chance_level=False` the first time, you can still call `disp.plot(plot_chance_level=True)` and get the chance level.


```suggestion
        prevalence_pos_label = Counter(y_true)[pos_label] / len(y_true)
```